gbpservice/nfp/service_vendor_agents/haproxy/haproxy-agent/etc/dhcp/dhclient-exit-hooks.d/haproxy_routing

```diff
@@ -26,16 +25,28 @@ function ip2dec () {
 all_interfaces=`ifconfig | grep "Link encap" | grep -v "127.0.0.1" |cut -d: -d " " -f 1`;
 index=0;
 for i in $all_interfaces; do
-    if [ "$i" != "lo" ] #&& [ "$i" != "eth0" ]
-    #if [ "$i" == "$interface" ]
+    if [ "$i" != "lo" ] 
     then
-        ip_addr=`ip addr show $i | grep "inet " | grep -v "/32" | awk -F'[/ ]+' '{ print $3 }'`
+        interface_ip_configure_time=30
+        while [[ $interface_ip_configure_time -gt 0 ]]; do
+                ip_addr=`ip addr show $i | grep "inet " | grep -v "/32" | awk -F'[/ ]+' '{ print $3 }'`
+                if [[ -z $ip_addr ]]; then
+                        sleep 5
+                else
+                        break
+                fi
+        ((interface_ip_configure_time-=5))
```

Comment:
Shows wrong indentation. Please check if it is like this in actual code or just shown like this in git. Though there is no functionality issue, code readability will be better.